### DISCUSSION TITLE
Tips for best trainings results - dataset size

### DISCUSSION THREAD

**Discussion by martingbl** [ultralytics/yolov5#12741] [Q&A] [answered]
Hello,
I would like to understand the recommended size of a dataset for training better.
Does the documentation refer to >=10000 Instances for training including or excluding validation/test data?
Because from my understanding training and validation are used during the training process but the recommeded instance amount could be refering only to the training dataset. Would be great to have more information on this.

Best regards, Taka

**Comment by jmarintur** [comment, marked as answer]
Hi @TheTaka96, the recommended size is for the whole [dataset](https://github.com/ultralytics/yolov5/wiki/Tips-for-Best-Training-Results#dataset). The dataset includes training, validation and test, hence, you should split it into three partitions. Note the importance of both, having at least 1,500 images per class and having minimum of 10,000 instances (labeled objects) per class.
